grow/common/oauth.py

```diff
@@ -83,6 +84,11 @@ def get_or_create_credentials(scope, storage_key=DEFAULT_STORAGE_KEY):
         credentials = tools.run_flow(flow, storage, flags)
         # run_flow changes the logging level, so change it back.
         logging.getLogger().setLevel(getattr(logging, 'INFO'))
+    # Avoid logspam by logging the email address only once.
+    if hasattr(credentials, 'id_token'):
+        email = credentials.id_token['email']
+        if _LAST_LOGGED_EMAIL != email:
```

Comment:
You aren't updating the `_LAST_LOGGED_EMAIL` anywhere?

Reply:
Oops. :) Fixed that.